src/dmclock_server.h

```diff
@@ -821,14 +831,23 @@ namespace crimson {
 	RequestTag tag(0, 0, 0, time);
 
 	if (!client.has_request()) {
-	  tag = RequestTag(client.get_req_tag(), client.info,
-			   req_params, time, cost);
+#ifndef DO_DYNAMIC_CLI_INFO_F
```

Comment:
As you recommended, I removed DO_DYNAMIC_CLI_INFO_F and used the boolean template parameter to the PriorityQueueBase class.